src/components/Filters/spec.js

```diff
@@ -133,6 +164,34 @@ describe('Filters component', () => {
       expect(component.find(FiltersSummary)).toHaveLength(0)
       component.unmount()
     })
+
+    it('should filter using "is" operator for both key and value properties of an object', () => {
```

Comment:
One last nitpick, these test names don't match what the test is doing, could you please change it to something like
```suggestion
    it('should  show the correct label when filtering using "is" operator for both key and value properties of an object', () => {
```